yt/visualization/plot_container.py

```diff
@@ -12,6 +12,7 @@
 #
 # The full license is in the file COPYING.txt, distributed with this software.
 #-----------------------------------------------------------------------------
+from yt import YTQuantity
```

Comment:
Could you change that to 
```suggestion
from yt.units import YTQuantity
```
?

Reply:
sure, thanks !